README

```diff
@@ -31,15 +26,15 @@ Tehnick has provided an up to date uhub PPA for Debian and Ubuntu based distribu
 
 ### Compile
 
-See doc/compile.txt document.
+See doc/compile.txt document or https://github.com/janvidar/uhub/blob/master/doc/compile.txt.
```

Comment:
Oh, sorry, I thought that this is `README.md` file (markdown-formated document) unlike plain-text documentation in `doc/` subdirectory.

@janvidar What do you think about converting of `README` to `README.md`?